cumulusci/core/tests/test_config_expensive.py

```diff
@@ -23,8 +24,13 @@
 
 @mock.patch("os.path.expanduser")
 class TestBaseGlobalConfig(unittest.TestCase):
-    def setUp(self):
-        self.tempdir_home = tempfile.mkdtemp()
+    @classmethod
+    def setUp(cls):
+        cls.tempdir_home = tempfile.mkdtemp()
+
+    @classmethod
+    def tearDownClass(cls):
+        shutil.rmtree(cls.tempdir_home)
```

Comment:
This doesn't look right; it is creating a new temp dir before every test case but only removing one of them (did you mean to use setUpClass instead of setUp?)